packages/core/src/ExecutorFactory.ts

```diff
@@ -4,15 +4,17 @@ import { Registry } from './Registry';
 import { UnfoldedDiagramFactory } from './UnfoldedDiagramFactory';
 import { Computer } from './types/Computer';
 import { Storage } from './types/Storage';
+import { InputObserverController } from './InputObserverController';
 
 export const ExecutorFactory = {
   create(
```

Comment:
As this is growing in terms of parameters, consider converting to named/keyed ({}).